src/plugins/walletConnectToDapps/v1/WalletConnectBridgeProvider.tsx

```diff
@@ -213,7 +215,9 @@ export const WalletConnectBridgeProvider: FC<PropsWithChildren> = ({ children })
     (err: Error | null, payload: { params: [{ peerMeta: IClientMeta }] }) => {
       if (err) moduleLogger.error(err, { fn: 'handleConnect' }, 'Error connecting')
       moduleLogger.info(payload, { fn: 'handleConnect' }, 'Payload')
-      setDapp(payload.params[0].peerMeta)
+      const dapp = payload.params[0].peerMeta
+      setDapp(dapp)
+      getMixPanel()?.track(MixPanelEvents.ConnectedTodApp, { dapp })
```

Comment:
At first I was going to comment LGTM even though we really only need description and url

<img width="402" alt="image" src="https://github.com/shapeshift/web/assets/17035424/f2260a88-7f1a-41d6-b03c-eb89b334bcfb">

But this won't serialize, Mixpanel works with nested properties, so if you track `dapp` as an object, this is how it will look

<img width="469" alt="image" src="https://github.com/shapeshift/web/assets/17035424/8cd98efd-f5ec-46f2-a061-013cac1e5720">

Apply one of these two and this will be happy

```suggestion
      getMixPanel()?.track(MixPanelEvents.ConnectedTodApp, dapp)
```

or

```suggestion
      // destructure me above
      getMixPanel()?.track(MixPanelEvents.ConnectedTodApp, { url, description })
```

Reply:
I think we'll profit from having `name` too (likely the one @willyogo wants to use for charts), and who knows whatever the dApps will include in the future with WalletConnect so let's keep `dapp`. Thanks for spotting my mistake before we collect useless data!